packages/pentaho/src/Canvas/ToolbarTabs/ToolbarTabs.styles.tsx

```diff
@@ -22,85 +23,80 @@ export const { staticClasses, useClasses } = createClasses(
       backgroundColor: theme.colors.atmo1,
       boxShadow: theme.colors.shadow,
       borderRadius: `0px 0px ${theme.radii.base} ${theme.radii.base}`,
-      gap: theme.space.sm,
       transition: "width 0.3s ease",
     },
     tabsContainer: {
-      position: "relative",
       display: "flex",
     },
     tabsList: {
       height: TAB_HEIGHT,
       background: theme.colors.atmo1,
+      borderEndStartRadius: theme.radii.base,
     },
     tab: {
-      width: `clamp(${MIN_TAB_WIDTH}px, 100%, ${MAX_TAB_WIDTH}px)`,
+      boxSizing: "border-box",
       border: `1px solid ${theme.colors.atmo1}`,
       borderBottom: "none",
       borderRadius: "10px 10px 0 0",
       boxShadow: "none",
       backgroundColor: theme.colors.atmo1,
+      "&:first-of-type": { borderEndStartRadius: theme.radii.base },
       [`&.${tabClasses.selected}`]: {
         color: theme.colors.primary,
-        backgroundColor: theme.colors.containerBackgroundHover,
+        backgroundColor: TAB_COLOR,
         borderColor: theme.colors.atmo4,
       },
+      [`&:hover:not(.${tabClasses.selected}), &:focus:not(.${tabClasses.selected})`]:
+        {
+          borderRadius: 0,
+          backgroundColor: TAB_COLOR,
+          borderColor: TAB_COLOR,
+          "&:first-of-type": { borderEndStartRadius: theme.radii.base },
+        },
+
+      // Hide icon when editor is hovered and focused
+      [`&:has($tabLabelEditor:hover) $tabIconContainer, &:has(.${toolbarTabEditorClasses.edit}) $tabIconContainer`]:
+        {
+          display: "none",
+        },
```

Comment:
The UI feel a bit janky as it's "jumping around" on hover - maybe we should only hide the icon when we're editing?

Reply:
As discussed, we'll talk with PD first 🙏